helpers/env-variables.ts

```diff
@@ -590,13 +590,6 @@ export const createFrontendEnvFile = async (
   },
 ) => {
   const defaultFrontendEnvs = [
-    {
-      name: "NEXT_PUBLIC_CHAT_API",
```

Comment:
Also need to remove the reference of this env variable?

Reply:
@leehuwuj we might want to keep this for this use case: https://github.com/run-llama/create-llama/pull/423#issuecomment-2492067519